modules/cactus-web/src/Table/Table.tsx

```diff
@@ -20,6 +20,7 @@ type CellAlignment = 'center' | 'right' | 'left'
 type CellType = 'th' | 'td'
 type BorderCorner = 'top-left' | 'top-right' | 'bottom-left' | 'bottom-right'
 export type TableVariant = 'table' | 'card' | 'mini'
+export type stickyColAlignment = 'right' | 'none'
```

Comment:
You should capitalize this to make it consistent with other types.
```suggestion
export type StickyColAlignment = 'right' | 'none'
```